.circleci/config.yml

```diff
@@ -1,216 +1,33 @@
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-guava:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-guava
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-hibernate-orm:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-hibernate-orm
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-spotbugs:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-spotbugs
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-spoon:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-spoon
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-spring-framework:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-spring-framework
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-hbase:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-hbase
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-Pmd-elasticsearch-lombok-ast:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-Pmd-elasticsearch-lombok-ast
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-exception-alot-of-projects:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-exception-alot-of-projects
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-warning-imports-guava:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-warning-imports-guava
-      - run:
-          command: ./.ci/validation.sh git-diff
-  no-warning-imports-java-design-patterns:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh no-warning-imports-java-design-patterns
-      - run:
-          command: ./.ci/validation.sh git-diff
-  sonarqube:
-    docker:
-      - image: *default-image
-    steps:
-      - checkout
-      - run:
-          command: |
-            ./.ci/validation.sh sonarqube
-      - run:
-          command: ./.ci/validation.sh git-diff
+commands:
+  run-script:
```

Comment:
Run_maven_script or something with maven in name 
as it deals with maven cache

Reply:
done